Doc/c-api/weakref.rst

```diff
@@ -88,6 +88,15 @@ as much as it can.
       Use :c:func:`PyWeakref_GetRef` instead.
 
 
+.. c:function:: int PyWeakref_IsDead(PyObject *ref)
+
+   Test if the weak reference *ref* is dead. Returns 1 if the reference is
+   dead, 0 if it is alive, and -1 with an error set if *ref* is not a weak
+   reference object.
```

Comment:
```suggestion
   reference object or ``NULL``.
```

Reply:
I would prefer to document "ref must not be NULL", rather than accepting NULL in the doc/spec.

Reply:
The proposed wording seems ambiguous to me. I think requiring that `ref` be a weak reference object covers the NULL case and we don't usually explicitly state that arguments must not be `NULL`.